frontend/public/module/k8s/k8s-ref.ts

```diff
@@ -1,7 +1,9 @@
+import { getReference } from '@console/dynamic-plugin-sdk/src/utils/k8s';
+
 // TODO(alecmerdler): Replace all manual string building with this function
 export const referenceForGroupVersionKind = (group: string) => (version: string) => (
```

Comment:
Maybe we also deprecate this so that we still push them to doing the right thing.

```
/**
 * @deprecated use getReference if absolutely needed, otherwise pivot towards using K8sGroupKindVersion objects where possible
 */
```

Reply:
:heavy_check_mark: